<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResultWidget</class>
 <widget class="QWidget" name="SearchResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tablewidget">
       <column>
        <property name="text">
         <string>操作</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>文件名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>大小</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>上传时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类型</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_38">
       <item>
        <widget class="QComboBox" name="show_num_comboBox">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
        border-radius: 1px;
        border: 1px solid #666666;
}

QComboBox::drop-down{
		subcontrol-position: top right;
		border-left-width: 1px;
        border-left-style: solid;
        
}
QComboBox::down-arrow{
	image: url(:/pic/resource/img/array_down.png);
}</string>
         </property>
         <property name="currentText">
          <string>100</string>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_47">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_48">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="the_left_pushButton">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 8pt &quot;楷体&quot;;</string>
              </property>
              <property name="text">
               <string>首页</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="to_left_pushButton">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="maximumSize">
               <size>
                <width>15</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>第</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="jumptopage_num_lineEdit_widget" native="true">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_20">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="jump_to_page_lineEdit_horizontalLayout"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="_label_">
              <property name="maximumSize">
               <size>
                <width>15</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>页</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="to_right_pushButton">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="the_right_pushButton">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 8pt &quot;楷体&quot;;</string>
              </property>
              <property name="text">
               <string>尾页</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="page_total_num_label">
              <property name="text">
               <string>共？页</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_17">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="metadata_totalnum_label">
         <property name="text">
          <string>显示1-20，共？条记录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
